### PR TITLE
[ticket/13132] Twig: Add the loops content to the root context

### PR DESCRIPTION
**Pull request by Nicofuma**
https://tracker.phpbb.com/browse/PHPBB3-13132

PHPBB3-13132

### CODE REVIEW

**Comment by Nicofuma** [comment]
Bump

File: tests/template/template_test.php [hunk 1]
```diff
@@ -359,6 +491,21 @@ public function template_data()
 				array(),
 				'inner_value',
 			),
+			// Disable: needs Twig 1.14.2
```
Comment (marc1706) [inline tests/template/template_test.php]: We have Twig 1.18 now so maybe uncomment these?

**Comment by Nicofuma** [comment]
updated (I will resend a PR against master)

**Comment by Nicofuma** [comment]
replaced by #3471

**Pull request closed without merging**